source/particle/property/integrated_strain.cc

```diff
@@ -0,0 +1,101 @@
+/*
+  Copyright (C) 2015 - 2016 by the authors of the ASPECT code.
+
+ This file is part of ASPECT.
+
+ ASPECT is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2, or (at your option)
+ any later version.
+
+ ASPECT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with ASPECT; see the file doc/COPYING.  If not see
+ <http://www.gnu.org/licenses/>.
+ */
+
+#include <aspect/particle/property/integrated_strain.h>
+
+namespace aspect
+{
+  namespace Particle
+  {
+    namespace Property
+    {
+      template <int dim>
+      void
+      IntegratedStrain<dim>::initialize_one_particle_property(const Point<dim> &,
+                                                              const Vector<double> &,
+                                                              const std::vector<Tensor<1,dim> > &,
+                                                              std::vector<double> &data) const
+      {
+        for (unsigned int i = 0; i < Tensor<2,dim>::n_independent_components ; ++i)
```

Comment:
I think in the beginning we just weren't sure if the strain tensor would remain symmetric, plus it was easier to implement for a full tensor. But it should be possible to use a symmetric tensor, I guess. For compositional fields that might even be more important than for the tracers (because we have to advect fewer fields). 
But like this, the implementations are the same for the tracers and compositional fields, which probably makes sense. If someone wants to use the finite strain in the material model, the same code can be used for both approaches. So if we change it here, we should probably change it for the fields too.